packages/cactus-plugin-ledger-connector-besu/src/test/typescript/integration/plugin-ledger-connector-besu/deploy-contract/get-block.test.ts

```diff
@@ -0,0 +1,63 @@
+import test, { Test } from "tape";
+import { v4 as uuidv4 } from "uuid";
+import { PluginRegistry } from "@hyperledger/cactus-core";
+import {
+  PluginLedgerConnectorBesu,
+  PluginFactoryLedgerConnector,
+  GetBlockV1Request,
+} from "../../../../../main/typescript/public-api";
+import { PluginKeychainMemory } from "@hyperledger/cactus-plugin-keychain-memory";
+import { BesuTestLedger } from "@hyperledger/cactus-test-tooling";
+import { LogLevelDesc } from "@hyperledger/cactus-common";
+import HelloWorldContractJson from "../../../../solidity/hello-world-contract/HelloWorld.json";
+import Web3 from "web3";
+import { PluginImportType } from "@hyperledger/cactus-core-api";
+
+test("can get block from blockchain", async (t: Test) => {
+  const logLevel: LogLevelDesc = "TRACE";
+  const besuTestLedger = new BesuTestLedger();
+  await besuTestLedger.start();
+
+  test.onFinish(async () => {
+    await besuTestLedger.stop();
+    await besuTestLedger.destroy();
+  });
+
+  const rpcApiHttpHost = await besuTestLedger.getRpcApiHttpHost();
+  const rpcApiWsHost = await besuTestLedger.getRpcApiWsHost();
+  const web3 = new Web3(rpcApiHttpHost);
+  const testEthAccount = web3.eth.accounts.create(uuidv4());
+
+  const keychainEntryKey = uuidv4();
+  const keychainEntryValue = testEthAccount.privateKey;
+  const keychainPlugin = new PluginKeychainMemory({
+    instanceId: uuidv4(),
+    keychainId: uuidv4(),
+    // pre-provision keychain with mock backend holding the private key of the
+    // test account that we'll reference while sending requests with the
+    // signing credential pointing to this keychain entry.
+    backend: new Map([[keychainEntryKey, keychainEntryValue]]),
+    logLevel,
+  });
+  keychainPlugin.set(
+    HelloWorldContractJson.contractName,
+    HelloWorldContractJson,
+  );
+  const factory = new PluginFactoryLedgerConnector({
+    pluginImportType: PluginImportType.Local,
+  });
+  const connector: PluginLedgerConnectorBesu = await factory.create({
+    rpcApiHttpHost,
+    rpcApiWsHost,
+    instanceId: uuidv4(),
+    pluginRegistry: new PluginRegistry({ plugins: [keychainPlugin] }),
+  });
+
+  const request: GetBlockV1Request = { blockHashOrBlockNumber: 0 };
+  const currentBlock = await connector.getBlock(request.blockHashOrBlockNumber);
```

Comment:
@twiggins120 

```suggestion
  const currentBlock = await connector.getBlock(request);
```

Reply:
@twiggins120 @ty-lazar I see this one got marked as resolved, but the code appears to be unchanged?

Reply:
@twiggins120 @petermetz It appears to be resolved now.